icd/api/pipeline_compiler.cpp

```diff
@@ -1499,6 +1499,7 @@ void PipelineCompiler::ApplyPipelineOptions(
     pOptions->shadowDescriptorTableUsage = Vkgc::ShadowDescriptorTableUsage::Enable;
     pOptions->shadowDescriptorTablePtrHigh =
           static_cast<uint32_t>(info.gpuMemoryProperties.shadowDescTableVaStart >> 32);
+    pOptions->enableRelocatableShaderElf = m_pPhysicalDevice->GetRuntimeSettings().enableRelocatableShaders;
```

Comment:
I have the use of the new pipeline option guarded by `LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 38`.  I don't know how the interface versions are suppose to work.  Let me know if it should be guarded by a higher version number or if we need a new version number.